docs(guides): add example for importing toml/yaml/json5 (#3945)
webpack/webpack.js.org: src/content/guides/asset-management.md

import { foo } from './data.json';
```

### Customize parser of JSON modules

> It's possible to import any `toml`, `yaml` or `json5` file as JSON module with a [custom parser](/configuration/module/#ruleparserparse) instead of specific webpack loader.

Reviewer: ```suggestion
It's possible to import any `toml`, `yaml` or `json5` files as a JSON module by using a [custom parser](/configuration/module/#ruleparserparse) instead of a specific webpack loader.
```

> Let's say you have a `data.toml`, a `data.yaml` and a `data.json5` under `src` folder:

Reviewer: ```suggestion
Let's say you have a `data.toml`, a `data.yaml` and a `data.json5` files under `src` folder:
```

__src/data.toml__

```toml
title = "TOML Example"

[owner]
name = "Tom Preston-Werner"
organization = "GitHub"
bio = "GitHub Cofounder & CEO\nLikes tater tots and beer."
dob = 1979-05-27T07:32:00Z
```

__src/data.yaml__

```yaml
title: YAML Example
owner:
  name: Tom Preston-Werner
  organization: GitHub
  bio: |-
    GitHub Cofounder & CEO
    Likes tater tots and beer.
  dob: 1979-05-27T07:32:00.000Z
```

__src/data.json5__

```json5
{
  // comment
  title: "JSON5 Example",
  owner: {
    name: "Tom Preston-Werner",
    organization: "GitHub",
    bio: "GitHub Cofounder & CEO\n\
Likes tater tots and beer.",
    dob: "1979-05-27T07:32:00.000Z"
  }
}
```

__webpack.config.js__

```javascript

> // make sure you have `toml`, `yamljs` and `json5` packages installed

Reviewer: move this commented out (grey) prerequisite up to where you mentioned about created file or to a place where you like as a separate paragraph

import { foo } from './data.json';
```

### Customize parser of JSON modules

It's possible to import any `toml`, `yaml` or `json5` files as a JSON module by using a [custom parser](/configuration/module/#ruleparserparse) instead of a specific webpack loader.

Let's say you have a `data.toml`, a `data.yaml` and a `data.json5` files under `src` folder:

__src/data.toml__

```toml
title = "TOML Example"

[owner]
name = "Tom Preston-Werner"
organization = "GitHub"
bio = "GitHub Cofounder & CEO\nLikes tater tots and beer."
dob = 1979-05-27T07:32:00Z
```

__src/data.yaml__

```yaml
title: YAML Example
owner:
  name: Tom Preston-Werner
  organization: GitHub
  bio: |-
    GitHub Cofounder & CEO
    Likes tater tots and beer.
  dob: 1979-05-27T07:32:00.000Z
```

__src/data.json5__

```json5
{
  // comment
  title: "JSON5 Example",
  owner: {
    name: "Tom Preston-Werner",
    organization: "GitHub",
    bio: "GitHub Cofounder & CEO\n\
Likes tater tots and beer.",
    dob: "1979-05-27T07:32:00.000Z"
  }
}
```

Install `toml`, `yamljs` and `json5` packages first:

```bash

> npm install toml --save-dev
> npm install yamljs --save-dev
> npm install json5 --save-dev

Reviewer: ```suggestion
npm install toml yamljs json5 --save-dev
```
